### ISSUE TITLE
Box quantity 

### ISSUE THREAD

**Issue by gpacker** [treeDiM/StackBuilder#69]
Anyway to get the number of cases, and the palletisation with just setting the number of boxes (inner product)?

**Comment by fgasnier71** [comment]
No. Not in this version. Should be offered in version 3 coming this year.

**Comment by gpacker** [comment]
Right, really nice that's a big step! can't wait for it!
Cheers!
